web/views.py

```
@@ -4352,6 +4457,140 @@ def streak_detail(request):
     return render(request, "streak_detail.html", {"streak": streak})
 
 
+@login_required
+@require_POST
+def track_social_share(request, course_slug):
```

Ruff (0.8.2)

Missing return type annotation for public function `track_social_share`

(ANN201)